tasks/devkit.rake

```diff
@@ -6,6 +6,7 @@ TDK = TerraformDevKit
 
 raise 'ROOT_PATH is not defined' if defined?(ROOT_PATH).nil?
 BIN_PATH = File.join(ROOT_PATH, 'bin')
+CONFIG_FILE ||= File.join(ROOT_PATH, 'config', 'config-%s.yml')
```

Comment:
Variables such as `CONFIG_FILE` (which may get their value injected from the top-level Rakefile) should be documented in the README.